src/buybutton.js

```diff
@@ -1,16 +1,18 @@
-import ShopifyBuy from 'shopify-buy/polyfilled';
+// import ShopifyBuy from 'shopify-buy/polyfilled';
+import ShopifyBuy, {Config} from 'shopify-buy';
 import UI from './ui';
 import productTemplates from './templates/product';
 
 window.ShopifyBuy = window.ShopifyBuy || ShopifyBuy;
+window.Config = window.Config || Config;
```

Comment:
You'll definitely want to namespace this global (`Config` is pretty likely to already exist on someone's page)